### PR TITLE
[DOCS] Adds enrich policies and data retention for Index Management

### PR DESCRIPTION
**Pull request by gchaps**
This PR updates the docs "Index Management in Kibana".  Included are:

- Data stream rentention values
- Enrich policies
- Updates to improve the flow of the doc

This PR addresses [this docs issue](https://github.com/elastic/platform-docs-team/issues/238).

Preview:
[Index Management in Kibana](https://elasticsearch_100922.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/index-mgmt.html)

### CODE REVIEW

File: docs/reference/indices/index-mgmt.asciidoc [hunk 1]
```diff
@@ -236,3 +215,31 @@ earlier.
...
+* The source indices that store enrich data as documents
+* The fields from the source indices used to match incoming documents
+* The enrich fields containing enrich data from the source indices that you want to add to incoming documents
+
+[role="screenshot"]
+image::images/index-mgmt/management-enrich-policies.png[Enrich policies]
+
+
+When creating an enrich policy, the UI walks you through the configuration setup and selecting the fields.
+Before you can use the policy with an enrich processor, you must execute the policy.
```
Comment (alisonelizabeth) [inline docs/reference/indices/index-mgmt.asciidoc:236]: Should we mention ES|QL here as well? cc @stratoula

**Review by alisonelizabeth — COMMENTED** [review]
Thanks for working on this, Gail! This is great. I left a few comments. Let me know what you think.

File: docs/reference/indices/index-mgmt.asciidoc (hunk 1 above)
Comment (abdonpijpelink) [reply docs/reference/indices/index-mgmt.asciidoc:236]: Doesn't hurt to mention it here too. Maybe something like:

```suggestion
Before you can use the policy with an enrich processor or {esql} query, you must execute the policy.
```

Comment (stratoula) [reply docs/reference/indices/index-mgmt.asciidoc:236]: I like it!!

File: docs/reference/indices/index-mgmt.asciidoc [hunk 2]
```diff
@@ -31,82 +25,67 @@ If you use {es} {security-features}, the following
...
-The *Data Streams* view lists your data streams and lets you examine or delete
-them.
+Investigate your data streams and address lifecycle management needs in the *Data Streams* view.
 
-To view more information about a data stream, such as its generation or its
-current index lifecycle policy, click the stream's name.
+The value in the *Indices* column indicates the number of backing indices. Click this number to drill down into details.
 
-To view and explore the data within a data stream, click the compass icon image:compassicon.png[width=3%] next to the data stream name to open {kibana-ref}/discover.html[Discover].
+A value in the data retention indicates that the data stream is managed by a <<data-stream-lifecycle,lifecycle policy>>.
```
[the remark this answers is not in the document]
Comment (abdonpijpelink) [reply docs/reference/indices/index-mgmt.asciidoc:64]: I'd say "data stream lifecycle policy" here to avoid confusion with index lifecycle policies.

**Review by abdonpijpelink — COMMENTED** [review]
Left a few suggestions.

File: docs/reference/indices/index-mgmt.asciidoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (alisonelizabeth) [reply docs/reference/indices/index-mgmt.asciidoc:64]: Are we referring to DSL configurations as policies, though? cc @tylerperk

File: docs/reference/indices/index-mgmt.asciidoc [hunk 3]
```diff
@@ -31,82 +25,67 @@ If you use {es} {security-features}, the following
...
+=== Manage indices
 
-Open {kib}'s main menu and click *Stack Management > Index Management*.
+Investigate your indices and perform operations from the *Indices* view.
 
 [role="screenshot"]
 image::images/index-mgmt/management_index_labels.png[Index Management UI]
 
-The *Index Management* page contains an overview of your indices.
+* To show details, perform operations such as close, forcemerge, and flush, or add a lifecycle policy use the *Manage* menu. This menu
```
[the remark this answers is not in the document]
Comment (yuliacech) [reply docs/reference/indices/index-mgmt.asciidoc:39]: The manage menu has the "Show index overview/settings/mapping" actions though

**Review by yuliacech — APPROVED** [review]

**Review by abdonpijpelink — APPROVED** [review]
LGTM too

**Review by alisonelizabeth — APPROVED** [review]
Latest LGTM. Thanks again @gchaps!

**Comment by abdonpijpelink** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|8.11|[](https://github.com/elastic/elasticsearch/pull/101151)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Pull request merged**